front/lib/utils.ts

```diff
@@ -13,6 +13,13 @@ export function new_id() {
   return Buffer.from(b).toString("hex");
 }
 
+export function client_side_new_id() {
+  // blake3 is not available in the browser
+  const u = uuidv4();
+  // return the last 10 characters of the uuid
```

Comment:
guaranteed to not have hyphens? Maybe we want to explicitely remove them here just to be explicit about it?